packages/ui-library/src/components/text-input/index.ts

```diff
@@ -14,7 +14,30 @@ import { BlrFormCaptionRenderFunction } from '../form-caption/renderFunction';
 import { BlrFormLabelRenderFunction } from '../form-label/renderFunction';
 import { BlrIconRenderFunction } from '../icon/renderFunction';
 import { TAG_NAME } from './renderFunction';
-
+import {
+  BlrBlurEvent,
+  BlrFocusEvent,
+  BlrSelectEvent,
+  BlrTextValueChangeEvent,
+  createBlrBlurEvent,
+  createBlrFocusEvent,
+  createBlrSelectEvent,
+  createBlrTextValueChangeEvent,
+} from '../../globals/events';
+
+export type BlrTextInputEventHandlers = {
+  blrFocus?: (event: BlrFocusEvent) => void;
+  blrBlur?: (event: BlrBlurEvent) => void;
+  blrTextValueChange?: (event: BlrTextValueChangeEvent) => void;
+  blrSelect?: (event: BlrSelectEvent) => void;
+};
+
+/**
+ * @fires blrFocus TextInput received focus
+ * @fires blrBlur TextInput lost focus
+ * @fires blrTextValueChange TextInput value changed
+ * @fires blrSelect Text in input got selected
```

Comment:
Can you also change "blrSelect Text in input got selected" to "blrSelect Text in TextInput got selected" for consistency.